jenkinsfiles/grant_access_to_app.groovy

```diff
@@ -1,28 +1,41 @@
 node {
 
+    properties([
+        parameters([
+            string(
+                name: 'APP_NAME',
+                description: 'Shiny App name',
+                defaultValue: ''),
+            string(
+                name: 'EMAILS',
+                description: 'Semicolon delimited list of email addresses',
```

Comment:
Just out of curiosity, why did you use semicolons (`;`) instead of commas (`,`) as email delimiters?